README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# calculatorapp
 
-## Available Scripts
+> The Math magician calc app solves simple caculations
 
-In the project directory, you can run:
+![screenshot]()
```

Comment:
- [ ] Please remove this meanwhile you have a better image to showcase your project so we don't showcase broken images or links in our Readme.